src/SFA.DAS.AssessorService.Web/Views/Certificate/Confirmation.cshtml

```diff
@@ -21,7 +21,7 @@
                         <br /><strong>@Model.GivenNames @Model.FamilyName</strong>
                     </div>
                     <div>
-                         @if (Model.SelectedGrade == CertificateGrade.Fail)
+                        @if (Model.SelectedGrade == CertificateGrade.Fail)
```

Comment:
The AC's state that the screen will change when it is NOT a fail - I'm not sure that is correct - it should probably change for both a pass and a fail - but the fail case has slightly different text - the implementation seems more correct than the AC's but they should match for testing - I think the AC's should state what happens for a pass and for a fail

Reply:
AC's updated, should not show new content for Fail grade however should display the new link